src/components/ThirdPartyCollectionDetailPage/ThirdPartyCollectionDetailPage.tsx

```diff
@@ -43,11 +104,51 @@ export default class ThirdPartyCollectionDetailPage extends React.PureComponent<
     return collection !== null && canSeeCollection(collection, wallet.address)
   }
 
+  areAllSelected(items: Item[]) {
+    const { itemSelectionState } = this.state
+    return items.every(item => itemSelectionState[item.id])
+  }
+
+  getSelectedItems() {
+    const { itemSelectionState } = this.state
+    return Object.values(itemSelectionState).filter(isSelected => isSelected)
+  }
+
+  isSearching() {
+    return this.state.searchText !== ''
+  }
+
+  search() {
```

Comment:
What do you think about changing the name of this method to something like:
```suggestion
  filterItemsBySearch() {
```

Reply:
Good idea, I was being a bit too minimalist :P